pkg/commands/exec.go

```diff
@@ -66,6 +66,10 @@ func RunExec(ctx CommandContext, args ExecArgs) error {
 		}
 	}
 
+	if server.PublicAddress.IP == "" && gateway == "" {
+		log.Warn("Your host hasn't gateway or public ip")
```

Comment:
`Your host has no public IP address, you should use '--gateway', see 'scw help exec'.`